packages/aggregator/src/analytics/aggregateAnalytics/adjustFetchOptionsToAggregation.js

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
Filename should probably have `List` on the end

Reply:
I had it like that initially but then omitted it, since two methods are exported:
* `adjustFetchOptionsToAggregationList`
* `adjustFetchOptionsToAggregationLists`

Happy to use `adjustFetchOptionsToAggregationList` though!